### ISSUE TITLE
Simplify a11y text on Nav block submenu indicators panel

### ISSUE THREAD

**Issue by getdave** [WordPress/gutenberg#68181]
## What problem does this address?

In the Navigation block settings, the `Submenu` section has two toggles. If they are both switched to "off" then following text appears as a notice/warning:

> The current menu options offer reduced accessibility for users and are not recommended. Enabling either "Open on Click" or "Show arrow" offers enhanced accessibility by allowing keyboard users to browse submenus selectively.

This is too verbose.

## What is your proposed solution?

Reduce the text to be simpler and hopefully shorter.

**Comment by AvulaRamSwaroop** [comment]
can you assign me this issue?

**Comment by carolinan** [comment]
Please note that this issue is not actionable until there is an agreement of what the new text string should be.

The current suggestions that I have copied from the two pull requests are:
- Reduced accessibility. Enable "Open on Click" or "Show arrow" for better keyboard navigation.
- 'The current menu options limit accessibility. Enabling "Open on Click" or "Show Arrow" improves usability for keyboard users navigating submenus.'


----
When asking to be assigned to an issue, and when creating a new pull request, please be mindful and check if someone is already working on it.
